test/backend/inference_backend.py

```diff
@@ -3126,6 +3121,7 @@ def get_test_models():
         },
         # ==OP== TopK
         # ==MIN== 10
+        # ==LIM== Unknown dimension in `X` of inputs is supported.
```

Comment:
This message does not seem to be a `limitation`. Could you make it clearer?

Reply:
Updated. Thanks!

Reply:
Did you push the change to the PR? Looks like it's unchanged.

Reply:
Updated. Thanks!